src/editor/Core/Commands.re

```diff
@@ -0,0 +1,43 @@
+type oniCommand = {
+  name: string,
+  command: unit => list(Actions.t),
+};
+
+type t = list(oniCommand);
+
+let oniCommands = [
+  {
+    name: "commandPalette.open",
```

Comment:
Thanks for starting up the `Commands` concept. It's a neat idea to have them map directly to our editor actions 👍